FreshRSS/FreshRSS
# Improved: Remove <li> as separator

In the dropdown menus there are seperators as `<li>` without any content.

So I cleaned the HTML and removed the empty `<li>`

Changes proposed in this pull request:

- templates (phtml)
- (S)CSS

How to test the feature manually:

1. open the dropdown menus
2. see the separator lines


Pull request checklist:
